pyamo/review.py

```diff
@@ -207,7 +227,7 @@ def __init__(self, parent, head, body):
 
     def _init_head(self, head):
         args = head.iterchildren().next().text.encode('utf-8').strip().split(" ")
-        _, self.version, _, month, day, year = filter(None, args)  # pylint: disable=bad-builtin
+        _, self.version, _, month, day, year = [_f for _f in args if _f]  # pylint: disable=bad-builtin
```

Comment:
I think we can remove disable=bad-builtin with this now, right?

Reply:
Don't know, I'm unfamiliar with pylint.